build.sbt

```diff
@@ -243,30 +127,22 @@ lazy val json = Project(id = "content-api-models-json", base = file("json"))
 
 lazy val benchmarks = Project(id = "benchmarks", base = file("benchmarks"))
   .dependsOn(json, scala)
-  .settings(commonSettings)
+  .settings(artifactProductionSettings)
   .enablePlugins(JmhPlugin)
   .settings(
     libraryDependencies += "com.google.guava" % "guava" % "19.0",
     Jmh / javaOptions ++= Seq("-server", "-Xms4G", "-Xmx4G", "-XX:+UseG1GC", "-XX:-UseBiasedLocking"),
     publishArtifact := false
   )
 
-lazy val npmBetaReleaseTagMaybe =
-  sys.props.get("RELEASE_TYPE").map {
-    case v if v == betaReleaseType =>
-      // Why hard-code "beta" instead of using the value of the variable? That's to ensure it's always presented as
-      // --tag beta to the npm release process provided by the ScroogeTypescriptGen plugin regardless of how we identify
-      // a beta release here
-      scroogeTypescriptPublishTag := "beta"
-
-    case v if v == snapshotReleaseType =>
-      scroogeTypescriptPublishTag := "snapshot"
-  }.toSeq
+lazy val npmPreviewReleaseTagMaybe = if (sys.env.get("RELEASE_TYPE").contains("PREVIEW_FEATURE_BRANCH")) {
```

Comment:
Note that the `RELEASE_TYPE` is coming from the environment variable supplied by `release.yml`: 

https://github.com/guardian/content-api-models/blob/d7be5bd72a958d1aa3aa0620c1a49c30a68eb484/.github/workflows/release.yml#L30-L33

The value of `RELEASE_TYPE` could be either `PREVIEW_FEATURE_BRANCH` or `FULL_MAIN_BRANCH`, as determined by [`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow/blob/d3089a063355b9dd5dcac0c9e2db3298053b209c/.github/workflows/reusable-release.yml#L35-L37)